Panic on multi user conference

Running into this error consistently when going from one to two participants in a room. This is using `pionwebrtc/ion-sfu:v0.4.4`

```
2020-06-05 15:25:47.064 INF
----------------rtc-----------------
pub: 1d2cdf3b-55cb-427a-84a5-75683e0adb2c
sub: e4530473-4945-458a-bbef-5f36088376a5

pub: dab890d9-842d-4d98-8519-8912d9073c27
sub: 565d9ebb-4872-49a7-927c-04cec4bcbdc0


2020-06-05 15:25:47.302 INF JitterBuffer.AddBuffer ssrc=3916934911
2020-06-05 15:25:47.303 INF NewBuffer BufferOptions={false 1000}
2020-06-05 15:25:47.303 INF JitterBuffer.WriteRTP buffer.SetSSRCPT(3916934911,96)
2020-06-05 15:25:47.369 INF Router got pli: [2582512119]
2020-06-05 15:25:47.391 INF Router got pli: [3916934911]
2020-06-05 15:25:47.502 INF Router got pli: [2582512119]
2020-06-05 15:25:47.531 INF Router got pli: [3916934911]
panic: crypto/cipher: output smaller than input

goroutine 1291 [running]:
crypto/cipher.(*ctr).XORKeyStream(0xc00025a320, 0xc0000a2024, 0x446, 0x446, 0xc00043ec0c, 0x454, 0x5a8)
	/usr/local/go/src/crypto/cipher/ctr.go:77 +0x286
github.com/pion/srtp.(*Context).encryptRTP(0xc00040c000, 0x0, 0x0, 0x0, 0xc0004143f0, 0xc00043ec0c, 0x454, 0x5a8, 0xc00039ce38, 0x55, ...)
	/go/pkg/mod/github.com/pion/srtp@v1.3.2/srtp.go:101 +0x22c
github.com/pion/srtp.(*SessionSRTP).writeRTP(0xc0003cc3c0, 0xc0004143f0, 0xc00043ec0c, 0x454, 0x5a8, 0x0, 0x0, 0x0)
	/go/pkg/mod/github.com/pion/srtp@v1.3.2/session_srtp.go:130 +0x102
github.com/pion/srtp.(*WriteStreamSRTP).WriteRTP(...)
	/go/pkg/mod/github.com/pion/srtp@v1.3.2/stream_srtp.go:124
github.com/pion/webrtc/v2.(*RTPSender).SendRTP(0xc0002f28c0, 0xc0004143f0, 0xc00043ec0c, 0x454, 0x5a8, 0x89043b, 0x0, 0xe771fe)
	/go/pkg/mod/github.com/pion/webrtc/v2@v2.2.13/rtpsender.go:180 +0x158
github.com/pion/webrtc/v2.(*Track).WriteRTP(0xc00034a1b0, 0xc0004143f0, 0xc00039cf30, 0x1)
	/go/pkg/mod/github.com/pion/webrtc/v2@v2.2.13/track.go:160 +0x141
github.com/pion/ion/pkg/rtc/transport.(*WebRTCTransport).WriteRTP(0xc00048a600, 0xc0004143f0, 0x1, 0x0)
	/go/src/github.com/pion/ion/pkg/rtc/transport/webrtctransport.go:456 +0x2ae
github.com/pion/ion/pkg/rtc.(*Router).subWriteLoop(0xc00016a000, 0xc0003c81e0, 0x24, 0xfddd80, 0xc00048a600)
	/go/src/github.com/pion/ion/pkg/rtc/router.go:144 +0xc5
created by github.com/pion/ion/pkg/rtc.(*Router).AddSub
	/go/src/github.com/pion/ion/pkg/rtc/router.go:211 +0x285
```

--- a/pkg/rtc/transport/rtptransport.go
+++ b/pkg/rtc/transport/rtptransport.go
@@ -291,16 +291,18 @@ func (r *RTPTransport) WriteRTP(rtp *rtp.Packet) error {
 		return err
 	}
 
+	pkt := *rtp
+
 	if rtp.SequenceNumber%10 == 0 {
 		r.idLock.Lock()
-		err := r.setIDHeaderExtension(rtp)
+		err := r.setIDHeaderExtension(&pkt)
 		if err != nil {
 			log.Errorf("error setting id to rtp extension %+v", err)
 		}
 		r.idLock.Unlock()
 	}
 
-	_, err = writeStream.WriteRTP(&rtp.Header, rtp.Payload)
+	_, err = writeStream.WriteRTP(&pkt.Header, rtp.Payload)
 
 	if err != nil {
 		log.Errorf("writeStream.WriteRTP => %s", err.Error())

--- a/pkg/rtc/transport/rtptransport.go
+++ b/pkg/rtc/transport/rtptransport.go
@@ -302,7 +302,7 @@ func (r *RTPTransport) WriteRTP(rtp *rtp.Packet) error {
 		r.idLock.Unlock()
 	}
 
-	_, err = writeStream.WriteRTP(&pkt.Header, rtp.Payload)
+	_, err = writeStream.WriteRTP(&pkt.Header, pkt.Payload)
 
 	if err != nil {
 		log.Errorf("writeStream.WriteRTP => %s", err.Error())

